runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
```

Comment:
`stdin`, `stdout`, and `stderr` only seem to be attached between the runtime and the application when process.terminal is false.  When it's true, the applications standard streams are attached to a TTY.  See [(1)][1].

[1]: https://github.com/opencontainers/specs/pull/87#discussion_r38173424